packages/icon-button/src/IconButton.js

```diff
@@ -33,8 +28,7 @@ export default class IconButton extends Component {
       onClick: this.props.onClick,
       onBlur: this.props.onBlur,
       onFocus: this.props.onFocus,
-      onMouseEnter: this.props.onMouseEnter,
-      onMouseLeave: this.props.onLeave
```

Comment:
This might've fallen off during rebase, but let's make sure `onMouseLeave` stays as a hook (https://github.com/Autodesk/hig/pull/814).

Reply:
😱